Readme.md

```diff
@@ -1,14 +1,13 @@
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T816800)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
 # Blazor Grid - Bind to a DataTable object
 
-The **DxGrid** component works with **IEnumerable** collections. That is why you cannot pass a **DataTable** object directly to the Grid's [Data](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.Data) property. 
+The **DxGrid** component supports binding to **DataTable** objects out of the box. You can generate such an object and pass it directly to the Grid's [Data](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.Data) property.

```

Comment:
This I wanted to avoid are "supports binding" (ING word) and "out of the box" (it's a fancy addition to the sentence that doesn't offer much value in this case). 

```suggestion
You can bind a **DxGrid** component to a **DataTable** object. Generate such an object and pass it directly to the Grid's [Data](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.Data) property.
```